src/main/java/com/ldtteam/domumornamentum/block/types/PillarShapeType.java

```diff
@@ -5,10 +5,13 @@
 
 public enum PillarShapeType implements StringRepresentable
 {
-    PILLAR_BASE("pillar_base"),
-    PILLAR_CAPITAL("pillar_capital"),
-    PILLAR_COLUMN("pillar_column"),
-    FULL_PILLAR("full_pillar");
+    /**
+     * these constants do not follow our naming conventions because forge does not allow the use of uppercase constants when creating an EnumProperty
+     */
+    pillar_base("pillar_base"),
+    pillar_capital("pillar_capital"),
```

Comment:
Caps was fine here, the problem

Reply:
the problem was? I'm gonna wait for your review to finish... didn't really think about the difficulty of trying to review while I'm making changes.

Reply:
weird, cutting off, the problem was the return of the name lower.

Reply:
got it. that makes sense.